wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -420,16 +423,23 @@ def _add_to_launch_queue(self, run: SweepRun) -> bool:
         elif _job is not None and _image_uri is not None:
             raise SchedulerError(f"{LOG_PREFIX}Sweep has both 'job' and 'image_uri'")
 
-        entry_point = None
-        if self._sweep_config.get("command"):
-            entry_point = Agent._create_sweep_command(self._sweep_config["command"])
+        args = create_sweep_command_args({"args": run.args})
+        entry_point = make_launch_sweep_entrypoint(
+            args, self._sweep_config.get("command")
+        )
+        if not entry_point:
+            launch_config = {"overrides": {"run_config": args["args_dict"]}}
+        else:
+            launch_config = {}
             wandb.termwarn(
                 f"{LOG_PREFIX}Sweep command {entry_point} will override"
                 f' {"job" if _job else "image_uri"} entrypoint'
             )
-
-        _args = _create_sweep_command_args({"args": run.args})["args_dict"]
-        launch_config = {"overrides": {"run_config": _args}}
+            if entry_point[0].startswith("${"):
```

Comment:
Should we capture the specific unresolved macros and tell users which ones we dont support. Ideally we then point them to the wandb launch sweeps page to see what macros are supported.